pyomo/contrib/fme/fourier_motzkin_elimination.py

```diff
@@ -68,6 +68,22 @@ def vars_to_eliminate_list(x):
             "Expected Var or list of Vars."
             "\n\tRecieved %s" % type(x))
 
+def _is_integer(x):
+    if int(x) == x:
```

Comment:
Is there any benefit of doing this within a tolerance?  Since you will raise an exception if this is False, I am thinking something along the lines of:
```python
def _to_integer(x):
    int_x = int(x)
    if abs(x-int_x) < int_tolerance:
        return int_x
    raise ValueError("%x is not an integer" % (x,))
```

Reply:
I don't know? But it's easy enough to do, and if you want to be really neurotic, you can set the tolerance to 0 (and not have a strict inequality), so yes, seems worth implementing.

Reply:
@jsiirola I took your suggestion, kind of, but I kept it returning a boolean so that I could have more specific error messages when I hit non-integer data. Though it seems a bit silly because I cast each of them twice that way...